docs/Displaying a User Interface/Popup Keyboards/index.md

```diff
@@ -177,10 +185,120 @@ KeyboardListener keyboardListener = new KeyboardListener() {
     public void onKeyboardDidSendEvent(KeyboardEvent event, String currentInputText) {
         // This is sent upon every event, such as keypresses, cancellations, and aborting
     }
+
+    @Override
+    public void onKeyboardDidUpdateInputMask(KeyboardEvent event) {
+        switch (event) {
+            case INPUT_KEY_MASK_ENABLED:
+                // The user enabled input key masking
+                break;
+            case INPUT_KEY_MASK_DISABLED:
+                // The user disabled input key masking
+                break;
+            default:
+                break;
+        }
+    }
 };
 ```
 !@
 
+@![iOS, android, javaSE, javaEE]
+### Configuring the Keyboard Properties
+You can change default keyboard properties by updating !@@![iOS]`sdlManager.screenManager.keyboardConfiguration`!@@![android, javaSE, javaEE]`sdlManager.getScreenManager().setKeyboardConfiguration()`!@@![iOS, android, javaSE, javaEE]. For example, you can set the keyboard layout to a `NUMERIC` keyboard and allow the user to mask the input characters.
```

Comment:
```suggestion
You can change default keyboard properties by updating !@@![iOS]`sdlManager.screenManager.keyboardConfiguration`!@@![android, javaSE, javaEE]`sdlManager.getScreenManager().setKeyboardConfiguration()`!@@![iOS, android, javaSE, javaEE]. 

#### Keyboard Layout (RPC 7.1+)
You can modify the keyboard layout by changing the keyboard configuration's `keyboardLayout`. For example, you can set a `NUMERIC` keyboard. It will default to `QWERTY` if not otherwise set.
```